apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/wpcom-nux.js

```diff
@@ -108,17 +106,15 @@ function getWpcomNuxPages( isGutenboarding ) {
 			alignBottom: true,
 		},
 		{
-			heading: __( 'Private until you’re ready', 'full-site-editing' ),
+			heading: __( 'Hidden until you’re ready', 'full-site-editing' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.